packages/containerd/containerd-cri-base-json

```diff
@@ -0,0 +1,151 @@
+{
+    "ociVersion": "1.0.2-dev",
+    "process": {
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "cwd": "/",
+{{#if settings.oci-defaults.capabilities}}
```

Comment:
A quick tip about handlebar templates, if you wrap the template with `~` like this, `{{~#if settings.oci-defaults.capabilities~}}`, you can then keep the indentation level you want. That would make reviewing these kind of changes much easier.

Example: https://github.com/bottlerocket-os/bottlerocket/blob/587f84bd30e39ceb227f0c3fd524bd66649438a5/packages/ecs-agent/ecs.config#L8-L10

Reply:
So, like this?

```
        {{~#if settings.oci-defaults.capabilities~}}
        "capabilities": {
            {{~oci_defaults settings.oci-defaults.capabilities~}}
        },
        {{~/if~}}
        {{~#if settings.oci-defaults.resource-limits~}}
        "rlimits": [
            {{~oci_defaults settings.oci-defaults.resource-limits~}}
        ],
        {{~/if~}}
```

Reply:
Yes. You got it